jobs/build/helm_sync/Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent any
+
+    parameters {
+        string(
+            name: "RPM_URL",
+            description: "Redistributable RPM URL. Example: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/helm/3.0.1/4.el7/x86_64/helm-redistributable-3.0.1-4.el7.x86_64.rpm",
+            defaultValue: ""
+        )
+        string(
+            name: "VERSION",
+            description: "Desired version name. Example: v3.0.1",
+            defaultValue: ""
+        )
+    }
+
+    stages {
+        stage("Validate params") {
+            if (!params.RPM_URL) {
+                error "RPM_URL must be specified"
+            }
+            if (!params.VERSION) {
+                error "VERSION must be specified"
+            }
+        }
+        stage("Download RPM") {
+            steps {
+                sh "wget --no-verbose ${params.RPM_URL} --output-document=helm-redistributable.rpm"
```

Comment:
`wget` will append `.<number>` to the download if the file already exists locally. Please guarantee that it does not yet exist.

Reply:
done: https://github.com/openshift/aos-cd-jobs/pull/2105/commits/15a3d0df6634e60de6eeae5826c6694bedfa97e5